activiti-cloud-query-service/activiti-cloud-starter-query/src/test/java/org/activiti/cloud/starter/tests/QueryTasksIT.java

```diff
@@ -1661,4 +1661,60 @@ public void should_getTask_when_queryFilteredByPriority() {
 
     }
 
+    @Test
+    public void shouldGetProcessInstancesFilteredByCompletedDate() {
```

Comment:
Test can be renamed as it is filtering Tasks and not ProcessInstances

Reply:
Good catch, thanks.  I will fix it.